File: represent/bench/java_flat/FormalBench_0236__FormalBenchHF__diverse__rule_1__IsOctagonal_llm_1.java
import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;

class IsOctagonal {

	public static int isOctagonal(int index) {
		int[] arr = new int[index + 1];
		arr[0] = 0;
		for (int j = 0; j <= index - 1; j++) {
			arr[j + 1] = arr[j] + 6 * j + 1;
		}
		return arr[index];
	}
}

File: represent/bench/c_flat/FormalBench_0236__FormalBenchHF__diverse__rule_1__IsOctagonal_llm_1.c
#include <stdlib.h>

int isOctagonal(int index);

int isOctagonal(int index) {

		int *arr = (int *)malloc(sizeof(int) * (index + 1));
int arr_len = index + 1;
		arr[0] = 0;
		for (int j = 0; j <= index - 1; j++) {
			arr[j + 1] = arr[j] + 6 * j + 1;
		}
		return arr[index];
}
